docs/agent/changelog.md

```diff
@@ -6,6 +6,20 @@ title: Changelog
 
 ## v3
 
+### ngrok Agent 3.15.0 - \[2024-08-29\]
+
+- Added `--url` agent CLI flag for simpler creation of HTTP(S), TLS, and TCP protocol endpoints.
+- Deprecated `--domain`, `--scheme`, and `--remote-addr` agent CLI flags, which have been replaced by `--url`.
+- Added `--metadata` and `--description` agent CLI flags when using `--url`.
+
+##### A new agent configuration format added to V3
```

Comment:
Do we use subheadings before in the changelog? I was thinking you could have nested bullets

Reply:
Originally had that, can change it back.